src/platforms/javascript/common/sourcemaps/tools/webpack.mdx

```diff
@@ -77,6 +77,8 @@ Set up the `SentryWebpackPlugin` as the last running plugin, otherwise, the resu
 
 If you prefer to upload source maps manually, configure Webpack to output source maps:
 
+<PlatformSection notSupported={["javascript.nextjs"]}>
```

Comment:
I like your thinking on this, but using PlatformSection is a little bit of an unusual approach. Generally, if we want to change a code sample, we build an `include` directory and pop the samples in there. If there's a sample for JS and a different sample that's specific to NextJS, that's fine. The JS one acts as the default for JS and its guides, and the NextJS code sample displays only for NextJS.

Let me know if you need help with this.

Reply:
Moved the samples to another directory, in `include/`. Not sure whether this is the best place though since it seems this directory contains content for all the languages and webpack is only for JS. Let me know what you think about it, and I can move the `PlatformSection` content above in a different PR.